### ISSUE TITLE
Item 32, p. 104 - BrokenDictionaryDB does nothing with __init__ parameters

### ISSUE THREAD

**Issue by yangmillstheory** [bslatkin/effectivepython#19]
I think the infinite recursion example directly following the class definition will still work, but is the intent of the constructor to set the incoming `data` parameter to `self._data`?

**Comment by bslatkin** [comment]
I think you mean this:

```
class BrokenDictionaryDB(object):
    def __init__(self, data):
        self._data = {}
```

Should instead be:

```
class BrokenDictionaryDB(object):
    def __init__(self, data):
        self._data = data
```

Is that right?

**Comment by bslatkin** [comment]
Fixed example code in 0fb46a6a4df8418f4e2a6ea6f07e983c887b5a09 and will be fixed in next printing.

**Comment by yangmillstheory** [comment]
Sorry for the (very) late reply, @bslatkin! Yes, that's exactly what I'd meant.
